doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,90 @@
+# Advocacy Ambassador program
+
+Node.js aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that the Node.js maintainers would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
```

Comment:
nit: this should link to the repository

Reply:
The repository does not exist yet.  After this lands and we create the repo I can add the link in later on.